dips/dip-2.md

```diff
@@ -44,18 +44,18 @@ The current roles in Diem are:
 | 1 |      DiemRoot      |      genesis       |         Globally          |  0xA550C18  |         N         |          -          |       N        |            3             |
 | 2 | TreasuryCompliance  |      genesis       |         Globally          |  0xB1E55ED  |         N         |          -          |       N        |            2             |
 | 3 |      Validator      |     DiemRoot      |  Per Association member   |      -      |         N         |          -          |       Y        |            1             |
-| 4 |  ValidatorOperator  |     DiemRoot      | At most one per Validator |      -      |         N         |          -          |       Y        |            1             |
+| 4 |  ValidatorOperator  |     DiemRoot      |  Per Validator            |      -      |         N         |          -          |       Y        |            1             |
```

Comment:
sent a message on this one as if this is operator : validator, it can still be 1 : M, even though today it is 1 : 1.